openwisp_monitoring/device/models.py

```diff
@@ -45,16 +45,18 @@ def __init__(self, *args, **kwargs):
     def data_user_friendly(self):
         if not self.data:
             return None
+        measured_at = datetime.strptime(self.measured_at[0:19], '%Y-%m-%dT%H:%M:%S')
```

Comment:
what does `self.measured_at` contain? Why is it being sliced?

Reply:
@nemesisdesign `influxdb` stores time like this `2016-06-13T17:43:50.1004002Z` where the value after the dot are nanoseconds. so i am cutting off the nanosecond to focus just on the time.